Makefile

```
@@ -206,6 +202,33 @@ allmetrics: html
 update: install
 	@. $(VENV); .sphinx/update_sp.py
 
+npm-install:
+	@command -v npm &> /dev/null
+	@if [ $? -eq 0 ]; then \
+	npm install ;\
+	else \
+	echo "Ensure npm is installed" ;\
+	fi
```

This doesn't work for me. Shows the following error:

```
(.venv) izmalk@Lenovo-Fortress-Lin2:~/test/sphinx/theme/ulwazi$ make npm-install
/usr/bin/npm
/bin/sh: 1: [: -eq: unexpected operator
Ensure npm is installed
(.venv) izmalk@Lenovo-Fortress-Lin2:~/test/sphinx/theme/ulwazi$ make vanilla-main
/usr/bin/npm
/bin/sh: 1: [: -eq: unexpected operator
Ensure npm is installed
echo "Compiling SCSS to CSS..."
Compiling SCSS to CSS...
/bin/sh: 1: [: Illegal number: npm-install
Error: sass not found. Please install sass first.
make: *** [Makefile:217: vanilla-main] Error 1
(.venv) izmalk@Lenovo-Fortress-Lin2:~/test/sphinx/theme/ulwazi$ 
```

I managed to make it work with the following change:

```suggestion
    @command -v npm &> /dev/null 2>&1 || { echo >&2 "Ensure npm is installed"; exit 1;} 
    npm install ;
```

I'm not sure which way is better and why the original doesn't work. Please advise.

---

It looks like it concatenates the make command, so the environment checked by `$?` is the previous shell command used rather than the `@command` line. Good fix, I'll adjust the make vanilla-main command too, as I think that also has issues.

Thanks!